api/base/serializers.py

```diff
@@ -1003,7 +1009,19 @@ def is_valid(self, clean_html=True, **kwargs):
         return ret
 
 
-class JSONAPISerializer(ser.Serializer):
+class PrefetchRelationshipsSerializer(ser.Serializer):
+    relationship_field_names = list()
```

Comment:
This probably shouldn't be defined on the class--it should not be shared across subclasses.